examples/simple/package.json

```diff
@@ -21,13 +21,5 @@
   "dependencies": {
     "tree-finder": "^0.0.13",
     "tree-finder-mockcontents": "^0.0.13"
-  },
-  "devDependencies": {
```

Comment:
Just for my information: Why are we moving the dev dependencies to the central package.json?

Reply:
we'll probably use these ones across multiple subpackages